qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -477,6 +477,48 @@ def test_scrub_split_dirfrags(self):
             timeout=30
         )
 
+    def test_scrub_remote_link(self):
+        """
+        test scrub remote link
+        """
+        test_dir_path = "test_dir"
+        self.mount_a.run_shell(["mkdir", test_dir_path])
+
+        file_dir_path = test_dir_path + "/file_dir"
+        self.mount_a.run_shell(["mkdir", file_dir_path])
+        file_path = file_dir_path + "/test_file.txt"
+        link_path = test_dir_path + "/test_link"
+        abs_link_path = "/" + link_path
+        self.mount_a.run_shell(["touch", file_path])
+        self.mount_a.run_shell(["ln", file_path, link_path])
+        file_ino = self.mount_a.path_to_ino(file_path)
+        dir_ino = self.mount_a.path_to_ino(file_dir_path)
+        rados_obj_file = "{ino:x}.00000000".format(ino=file_ino)
+        rados_obj_dir = "{ino:x}.00000000".format(ino=dir_ino)
+        self.fs.flush()
+        time.sleep(20)
+        self.fs.rados(["rm", rados_obj_file], pool=self.fs.get_data_pool_name())
+        self.fs.rados(["rm", rados_obj_dir], pool=self.fs.get_metadata_pool_name())
+        self.fs.fail()
+        time.sleep(10)
+        self.fs.set_joinable()
```

Comment:
All this is doing is to startup the MDS with an empty cache. Why not failover the MDS in that case?